library/src/skins/default/src/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -6,6 +6,11 @@
     margin-left: 20px;
   }
 
+  .text-bg-info {
+    background-color: var(--sakai-primary-color-1) !important; 
+    color: var(--sakai-text-color-inverted) !important;
```

Comment:
this breaks the cascade how about something like:
`
$info: var(--sakai-primary-color-1);
$info-bg-subtle: var(--sakai-text-color-inverted);
$info-text-emphasis: var(--sakai-primary-color-1);
$info-text: var(--sakai-primary-color-1);
`

Reply:
thanks @ern